test/loader/test_neighbor_sampler.py

```diff
@@ -9,23 +10,32 @@
 def test_neighbor_sampler():
     torch.manual_seed(12345)
     edge_index = erdos_renyi_graph(num_nodes=10, edge_prob=0.5)
+    sparse_edge_index = SparseTensor.from_edge_index(edge_index)
     E = edge_index.size(1)
 
     loader = NeighborSampler(edge_index, sizes=[2, 4], batch_size=2)
+    sparse_loader = NeighborSampler(sparse_edge_index, sizes=[2, 4],
+                                    batch_size=2, collate_fn="test_init",
+                                    dataset="test_init")
     assert loader.__repr__() == 'NeighborSampler(sizes=[2, 4])'
     assert len(loader) == 5
+    device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
```

Comment:
```suggestion
```
Lets drop device.

Reply:
This is because I want to test  `edge_index.to(device)`?

Reply:
We don't need to test if code works on different devices.